.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+<!-- Thank you for submitting a Pull Request. Please: 
+* Read our Pull Request guidelines:
+  https://github.com/github/vscode-codeql/blob/master/CONTRIBUTING.md#submitting-a-pull-request.
+* Update [CHANGELOG.md](../extensions/ql-vscode/CHANGELOG.md) if you've made any user-visible changes.
```

Comment:
I suggest you pull this out of the comment so it's visible, and make it a "Changelog updated" checkbox.

Reply:
Good idea, thanks!